packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -847,15 +824,17 @@ class MultiSelectComboBox extends ResizeMixin(InputControlMixin(ThemableMixin(El
 
   /** @private */
   __getOverflowWidth() {
-    const chip = this.$.overflow;
+    const chip = this._overflow;
 
     chip.style.visibility = 'hidden';
     chip.removeAttribute('hidden');
 
     // Detect max possible width of the overflow chip
-    chip.setAttribute('part', 'chip overflow');
+    const label = chip.label;
+    chip.label = '3';
```

Comment:
What we do here is to force using `3` as a label to ensure the overflow chip has max possible width, e.g.

![Screenshot 2022-10-25 at 15 12 52](https://user-images.githubusercontent.com/10589913/197769971-52e85e66-c7b8-4a90-a3b3-abc8c4174790.png)

Previously, this was done using `part` but now we use `label` attribute instead.

Reply:
Maybe add a suffix comment like `// Measure with widest number character`?

Also not sure if it's relevant here that the number of overflow items can reach two digits or more?

Reply:
Thanks, makes sense. I will update that, and maybe set to `10` indeed to ensure space is enough.

Reply:
Done. Changed to set `count` attribute to `99` which seems to indicate the purpose a bit cleaner.